deployment_cdk/_layout_deployment.adoc

```diff
@@ -44,7 +44,7 @@ endif::production_build[]
 
 == Deployment options
```

Comment:
Since this section is now actionable (instead of simply descriptive as in Docs 2.0), I suggest a task-type heading to indicates this: _Choose a deployment option_.

This note also applies to the CFN, EKS, and terraform versions of `_layout_deployment.adoc`.